cfg/2.0.0/definitions.yaml

```diff
@@ -7033,13 +7033,16 @@ groups:
       description: "Ensure rsyslog default file permissions configured"
       sub_checks:
       - check:
-          audit: "grep ^\\$FileCreateMode /etc/rsyslog.conf /etc/rsyslog.d/*.conf"   
+          audit: "grep ^\\$FileCreateMode /etc/rsyslog.conf /etc/rsyslog.d/*.conf -h | tr \"$\" \" \" "   
```

Comment:
Removed file name and $ from the start of the output

Reply:
I would like to try this. I have a vagrant VM with a `/etc/rsyslog.conf` available. And this is what I see:

```
vagrant@vagrant:~$ grep ^\\$FileCreateMode /etc/rsyslog.conf -h | tr \"$\" \" \"
tr: extra operand ‘"’
Try 'tr --help' for more information.
grep: Trailing backslash
vagrant@vagrant:~$ grep ^\\$FileCreateMode /etc/rsyslog.conf -h | tr \"$\" \"
grep: Trailing backslash
vagrant@vagrant:~$ grep ^\\$FileCreateMode /etc/rsyslog.conf -h
grep: Trailing backslash
```

What am I doing wrong?

Reply:
There is escaping, remove the back slash before special characters:
`grep ^\$FileCreateMode /etc/rsyslog.conf /etc/rsyslog.d/*.conf -h | tr "$" " "`

Reply:
OK, I'll have to take your word for it.